build.gradle

```diff
@@ -139,6 +139,7 @@ allprojects {
 	    force "org.jetbrains.kotlin:kotlin-stdlib-common:" + kotlinVersion
 	    force "org.jetbrains.kotlin:kotlin-stdlib:" + kotlinVersion
 	    force "org.yaml:snakeyaml:" + snakeYamlVersion
+            force "com.fasterxml.woodstox:woodstox-core:" + woodstoxVersion
```

Comment:
Why do we need this new dependency? Just to force it to a version that does not contain CVE? What dependency is transitively importing this one?

Reply:
Azure is bringing this woodstox as a transitive dependency and it is present in pravega/lib folder as well. Yes just to force it to a version that doesn't contain a CVE.